app/views/deploys/new.html.erb

```diff
@@ -24,6 +24,17 @@
 
       <%= Samson::Hooks.render_views(:deploy_form, self, project: @project, form: form) %>
 
+      <% if @recent_releases %>
```

Comment:
I think you do, because if you just create a new project with no releases, it'll crash (can't call each on nil). The specs were crashing before I added this check.